string grasp_type
int16 grasp_step
int16 grasp_force
